modules/spring-config/src/main/resources/META-INF/mule.xsd

```diff
@@ -1221,6 +1217,18 @@
         </xsd:restriction>
     </xsd:simpleType>
 
+    <xsd:simpleType name="extendedNotificationTypes">
+        <xsd:restriction base="xsd:NMTOKEN">
+            <xsd:pattern value="[a-z]+:[A-Z\-]+">
+                <xsd:annotation>
+                    <xsd:documentation>
+                        Allows plugins to define their own notifications, in format "[plugin]:[NOTIFICATION]". 
```

Comment:
Not sure if there will be other artifact providing notifications... but what aoubt using "artifactID" instead of "plugin"?